pkg/ccl/changefeedccl/sink.go

```diff
@@ -111,6 +111,7 @@ type EventSink interface {
 		key, value []byte,
 		updated, mvcc hlc.Timestamp,
 		alloc kvevent.Alloc,
+		headers map[string][]byte, // TODO: should this be kv pairs instead?
```

Comment:
it seems like k-v pairs might be a more desirable representation. what do you guys think?

Reply:
What would that look like?